public/assets/styles/_sections/Intro.less

```diff
@@ -0,0 +1,116 @@
+@top-spacing: 100px;
+
+#Intro{
+  padding-bottom: 2em;
+  #Logo {
+    display: flex;
+    margin: 2em auto;
+    justify-content: center;
+    font-weight: 100;
+    font-size: 1.5em;
+    letter-spacing: 5px;
+    img{
+      margin-right: 20px;
+    }
+  }
+  .content {
+    width: 90%;
+    margin: 0 auto;
+    text-align: center;
+    h2 {
+      font-size: 33px;
+      font-weight: normal;
+      letter-spacing: -1px;
+      line-height: 50px;
+
+    }
+    p {
+      font-size: 18px;
+      letter-spacing: 0;
+      color: @light-gray;
+      line-height: 30px;
+    }
+
+    .button {
+      margin: 3em 0;
+    }
+  }
+
+  .phones {
```

Comment:
There is a slight issue here on some wider "mobile" sizes:

<img width="668" alt="screenshot 2017-08-19 02 16 54" src="https://user-images.githubusercontent.com/3491886/29480907-b0a03bd6-8484-11e7-888d-8e1fa88a1509.png">

The solution is to use max-width. I often use max-width to help control fluid-width elements. For example, I would add:
```
max-width: 400px;
margin: 0 auto;
```
This will make sure the the phone assets never get too big, and that they will always be centered.